test/common/include/alpaka/test/acc/TestAccs.hpp

```diff
@@ -108,16 +108,29 @@ namespace alpaka
                     typename TIdx>
                 using AccCpuOmp2ThreadsIfAvailableElseInt = int;
 #endif
-#if defined(ALPAKA_ACC_CPU_BT_OMP4_ENABLED) && !defined(ALPAKA_CUDA_CI)
+#if defined(ALPAKA_ACC_ANY_BT_OMP5_ENABLED) && !defined(TEST_UNIT_KERNEL_KERNEL_STD_FUNCTION) \
+        && !( BOOST_COMP_GNUC && ( \
+                /* tests excluded because of GCC10 Oacc / Omp5 target symbol bug with multiple units */ \
```

Comment:
For example, the intrinsic test produces the following linker errors (both with OpenMP5 target and OpenACC, the example below is OpenACC):
```
`_ZNK6alpaka6kernel14TaskKernelOaccISt17integral_constantImLm1EElNS_3mem4view12MemSetKernelEJRKhPhRNS_3vec3VecIS3_lEESD_EEclERKNS_3dev7DevOaccE._omp_fn.0' referenced in section `.gnu.offload_funcs' of CMakeFiles/intrinsic.dir/src/Popcount.cpp.o: defined in discarded section `.text._ZNK6alpaka6kernel14TaskKernelOaccISt17integral_constantImLm1EElNS_3mem4view12MemSetKernelEJRKhPhRNS_3vec3VecIS3_lEESD_EEclERKNS_3dev7DevOaccE._omp_fn.0[_ZNK6alpaka6kernel14TaskKernelOaccISt17integral_constantImLm1EElNS_3mem4view12MemSetKernelEJRKhPhRNS_3vec3VecIS3_lEESD_EEclERKNS_3dev7DevOaccE]' of CMakeFiles/intrinsic.dir/src/Popcount.cpp.o
```
There are two compilarion units in this order:
1. Ffs.cpp.o
2. Popcount.cpp.o

MemSetKernel is used in both, the symbols are missing for the second unit. They are always missing for all but the first unit.

I just tested this with a fresh gcc master build. There it appears fixed, so we can expect this to be fixed in GCC 11. Adding another check... .

Reply:
New check: https://github.com/alpaka-group/alpaka/pull/1126/commits/d20db3eeb5bc541aa7e234bb3ea273e2f0e2d489

Reply:
Do we need this whole test disabling stuff in this PR at all? We are not enabling any gcc builds so we can simply wait for gcc-11 and only enable it there.

Reply:
The same stuff is in the OpenACC PR, where it is needed, because there is currently no other option than gcc10 in the CI for this one. So, it is mostly there for symmetry, although one could try to enable this with gcc10 in the CI, too. The reason I did not is because I can only compile but not run on my local machine. This is because it only compiles with `-foffload=disable` (i.e. no GPU) which will make it generate code for the host only but at runtime it sees a GPU and allocates the memory there, making the tests segfault. In the CI there is no GPU, so it will either work, because it allocates on the host or also fail, because it fails to allocate anything in target. I did not test this, but wanted to keep the option open.

On the other hand: could we make a compile-only Ci job?

Reply:
Yes, currently we have the variable `ALPAKA_CI_ANALYSIS` which selects whether the alpaka tests should be run or if the self-contained-header build (and some other small things) should be done. This was necessary in Travis CI because both things combined took too long. This may not be necessary now with github actions but this is still available.

Reply:
I just took a look at the gcc release dates and it lloks like gcc-11 will not be released before May 2021. So waiting for it does not seem to be an option.

Reply:
I added an `ALPAKA_CI_ANALYSIS` job with gcc 10 which includes OMP5.